### DISCUSSION TITLE
IPv6

### DISCUSSION THREAD

**Discussion by cssmith4** [vim/vim#13459] [Q&A]
Hello Vim Community,

Does anyone know where I can find any documentation that states that Vim supports IPv6?  I know it does, but looking for some literature or link that states this is the case.

Thank you,

Shane

**Comment by yegappan** [comment]
The only documentation that states the IPv6 support is in [`:help channel-address`](https://vimhelp.org/channel.txt.html#channel-address).
You can use the `:helpgrep IPv6` command to locate the places where IPv6 is mentioned in the Vim documentation.

**Reply by cssmith4** [reply]
Can we create a markdown page stating Vim's overall IPv6 conformity?  It may be as simple as adding a page that states the supported Operating Systems, IPv6, etc.  A matrix format could be very helpful.

**Reply by chrisbra** [reply]
what for? It is already in the help. Other than for the channels, Vim doesn't do networking. So why do you need this?
